apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/testing/vulnerability_report/VulnerabilityReport.jsx

```diff
@@ -0,0 +1,535 @@
+import { useEffect, useRef, useState } from 'react';
+import { useParams } from 'react-router-dom';
```

Comment:
break the file, so that it becomes easier to debug, abstract out the functions to a different file